lib/cretonne/src/ir/dfg.rs

```diff
@@ -106,6 +106,23 @@ impl DataFlowGraph {
     }
 }
 
+/// Resolve value aliases.
+///
+/// Find the original SSA value that `value` aliases.
+fn resolve_aliases(values: &EntityMap<Value, ValueData>, value: Value) -> Value {
```

Comment:
I compared the generated release-build code before and after making this change, and it doesn't seem significantly different. Since other places in the code find the current return-the-new-value interface more convenient, my sense is to leave it as is for now.